jekyll/_cci2/vs-code-extension-overview.adoc

```diff
@@ -165,12 +165,50 @@ Both of these commands can be run by:
 
 [#open-source-language-server]
 === Open source language server
 
-The config helper is based on a dedicated language server specific for CircleCI YAML files, which is open source. You can view its source code, contribute and add issues directly on the project repository: link:https://github.com/CircleCI-Public/circleci-yaml-language-server[circleci-yaml-language-server].
+The config helper is based on a dedicated language server specific for CircleCI YAML files, which is open source. You can view its source code, contribute and add issues directly on the project repository: link:https://github.com/CircleCI-Public/circleci-yaml-language-server[CircleCI YAML language server].
 
 You can also integrate the language server into any editor which supports the Language Server Protocol, and build your own plugin to benefit from config helper capabilities in your favourite editor.
 
+[#trigger-a-config-test-run]
+== Trigger a config test run
+
+NOTE: The config test run feature is currently in **closed preview**. Config test runs are available on the **Free Plan**, for accounts integrated through the **GitHub OAuth app**, or **Bitbucket**. To find out which GitHub account type you have, see the xref:github-integration#[GitHub OAuth app integration] page.
+
+Use config test runs to iterate on your pipeline configuration without committing your trial and error changes to your version control system. Run and validate your full pipeline, or select jobs and workflows to validate individually. View the results of your config test runs in the CircleCI web app, just the same as any other pipeline.
+
+[#prerequisites]
+=== Prerequisites
+
+* v2.0 or higher of the CircleCI VS Code extension
+* A CircleCI account integrated through the GitHub OAuth app, or Bitbucket. To find out which GitHub account type you have, see the xref:github-integration#[GitHub OAuth app integration] page
+* You must be on the link:https://circleci.com/pricing/[Free plan]
+
+[#steps]
+=== Steps
+
+[WARNING]
+====
+Triggering a pipeline with a _custom_ configuration can have side effects, for example, unwanted deployment. We recommend using this feature **only** on feature branches.
+
+Config test run pipelines have access to contexts, and project environment variables, based on which branch they are run on, and user permissions
+====
+
+NOTE: Config test runs are disabled for projects that make use of xref:dynamic-config#[dynamic configuration]
+
+The following steps show how to trigger a config test run pipeline from the VS Code extension:
+
+. Open VS Code and select the CircleCI logo in the sidebar. The **Pipelines** and **Run** panels will be visible.
+. Expand the Run panel to view your projects.
+. For each project, there is a button to trigger the config test run, and check boxes for each workflow and job for you to choose which parts of your config you want to run. Select your options, and click icon:play[].
```

Comment:
I would maybe add a step that makes it clear that any local change to the config will be applied. Something like:
* Any local change you make to your configuration file will be included in the test run